types/shared.d.ts

```diff
@@ -176,6 +176,8 @@ export interface StripeConnectInstance {
    * @returns A promise that resolves when the user is logged out.
    */
   logout: () => Promise<void>;
+
+  debugInstance: () => Promise<StripeConnectInstance | undefined>;
```

Comment:
We don't need to add this to the .d.ts! This is only meant for internal integrations.